app/models/data_update_script.rb

```diff
@@ -0,0 +1,35 @@
+class DataUpdateScript < ApplicationRecord
+  DIRECTORY = Rails.root.join("lib/data_update_scripts").freeze
```

Comment:
Is there a possiblity that scripts get unaligned with rows in the DB?

Reply:
There is definitely that risk if we have a script that updates a column and then in. The future we say remove that column. In that case you would also want to remove the script. Ideally the script would have a basic test and that would fail if the column was removed.